### ISSUE TITLE
[Magiclysm] Monster meat purified with Cauldron of Purification cannot be smoked in stationary smokers

### ISSUE THREAD

**Issue by Jan-Blasiak** [CleverRaven/Cataclysm-DDA#52960]
### Describe the bug

Monster meat purified with Cauldron of Purification cannot be smoked in stationary smokers

### Steps To Reproduce

Purify mutant meat in CoP, try to put into a stationary smoker.

### Expected behavior

Able to smoke the meat in stationary smokers.

### Screenshots

![Zrzut ekranu (466)](https://user-images.githubusercontent.com/61629226/142932022-62ce5a89-7c37-4593-bc7c-898d22b1e7b2.png)

### Versions and configuration

03f9cc1 (tiles)

**Comment by Maleclypse** [comment]
Does the stationary smoker use a meat list?

**Comment by Maleclypse** [comment]
```  if( portable ) {
            smenu.addentry_desc( 2, !full_portable, 'f',
                                 full_portable ? _( "Insert food for smoking… smoking rack is full" ) :
                                 string_format( _( "Insert food for smoking… remaining capacity is %s %s" ),
                                                format_volume( remaining_capacity_portable ), volume_units_abbr() ),
                                 _( "Fill the smoking rack with raw meat, fish or sausages for smoking or fruit or vegetable or smoked meat for drying." ) );
```
I can't figure out where it sets what items can go into this.

**Comment by Maleclypse** [comment]
Fixed by #61145
